qtpy/QtCore.py

```diff
@@ -78,11 +78,6 @@
 elif PYSIDE2:
     from PySide2.QtCore import *
 
-    try:  # may be limited to PySide-5.11a1 only
-        from PySide2.QtGui import QStringListModel
-    except Exception:
-        pass
-
```

Comment:
Was this change not done in #407 ? Maybe something happened when updating/merging the local branch with upstream or the remote branch? @CAM-Gerlach you know if this should prevent us from merging this as it is or maybe with the squash merge is not important?

Reply:
Yeah, I noticed that; not 100% sure of the exact chain of events without doing a deep-dive investigation, but something went wrong when the contributor merged with the remote branch on their fork, which is messing up the merge-base used for the three-way diff on the PR (as that change was indeed applied to ``master``). I'm almost certain it will vanish when squash-merged and not cause any issues, since all the commits are squashed and that new commit is rebased against the latest ``master``, which is why I didn't say anything before, and if there was a problem GitHub would not allow the merge.

Reply:
And yup, looked like it indeed worked as expected :tada: